xborders

```diff
@@ -384,6 +392,11 @@ class Highlight(Gtk.Window):
 
     def _calc_border_geometry(self, window):
         x, y, w, h = window.get_geometry()
+        
+        w /= self.scale
+        h /= self.scale
+        x /= self.scale
+        y /= self.scale
```

Comment:
When I hear scale I imagine an 0..n value that is multiplied? So why are we dividing